src/convertToObject.js

```diff
@@ -12,7 +12,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+ const newObject = {};
+
+  const filterString = sourceString.split(';');
+
+  for (let i = 0; i < filterString.length; i++) {
```

Comment:
Instead of using a standard for loop, use a for...of loop which is more concise and easier to read. Or you can think about array methods.